bin/decode_dist.R

```diff
@@ -149,6 +149,16 @@ main <- function(opts) {
   results_csv_path <- file.path(opts$output_dir, 'results.csv')
   write.csv(res$fit, file = results_csv_path, row.names = FALSE)
 
+  # Write residual histograph as a png.
+  results_png_path <- file.path(opts$output_dir, 'residual.png')
+  png(results_png_path)
+  breaks <- pretty(res$residual, n = 200)
+  step <- breaks[2] - breaks[1]  # distance between bins
+  histogram <- hist(res$residual, breaks, plot = FALSE)
+  histogram$counts <- histogram$counts / sum(histogram$counts)  # convert the histogram to frequencies
+  plot(histogram, main = "Histogram of the residual")
```

Comment:
change xlab here: "Residual (observed - model predicted, m x k values)"

or that can be in the title.